src/pages/index.js

```diff
@@ -78,19 +104,129 @@ editProfileButton.addEventListener("click", () => {
   profilePopup.open();
 });
 
-//add new card
+profileImageElement.addEventListener("click", () => {
+  imageChangePopup.open();
+});
+
+/*////////////////////////////////////////////////////////////////////
+/                      Popups                                        /
+////////////////////////////////////////////////////////////////////*/
+
+const viewImagePopup = new PopupWithImage(".view-image-popup");
+const confirmDeletePopup = new PopupWithForm({
+  popupSelector: ".delete-card-popup",
+  handleFormSubmit: () => {
+    deleteCard(cardElement);
+  },
+});
 
 const addCardPopup = new PopupWithForm({
   popupSelector: ".add-card-popup",
   handleFormSubmit: (formData) => {
-    renderCard(formData);
-    addCardPopup.close();
+    addCardPopup.setSavingMessage();
+    api
+      .addCard(formData)
+      .then((data) => {
+        renderCard(data);
+        addCardPopup.close();
+        addCardPopup.resetSubmitMessage("Create");
+      })
+      .catch((err) => {
+        console.error(`Error: ${err}`);
+      });
   },
 });
 
-const addLocationButton = document.querySelector(".profile__add-button");
+const profilePopup = new PopupWithForm({
+  popupSelector: ".profile-popup",
+  handleFormSubmit: () => {
+    const data = {
+      name: editProfileName.value,
+      about: editProfileSubtitle.value,
```

Comment:
Only method `_getInputValues` collects all the data of the inputs and passes it to the call of `submitForm`. So you need to use the collected data rather than the inputs.